app.go

```diff
@@ -749,6 +752,21 @@ func (app *App) Done() <-chan os.Signal {
 	return c
 }
 
+func (app *App) Wait() <-chan ShutdownSignal {
+	c := make(chan ShutdownSignal, 1)
+
+	app.waitsMu.Lock()
+	defer app.waitsMu.Unlock()
+
+	if app.shutdownSignal != nil {
+		c <- *app.shutdownSignal
+		return c
```

Comment:
Let's add a quick comment here indicating what this special case represents.

```suggestion
	if app.shutdownSignal != nil {
		// Already shut down.
		c <- *app.shutdownSignal
		return c
```